webpack/ForemanTasks/Components/TasksDashboard/Components/TasksTimeRow/TasksTimeRow.stories.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
@sharvit - using hooks outside of a component gives an error, transforming this to `WithState` solve it